src/d3a/models/market/market_structures.py

```diff
@@ -114,93 +137,62 @@ def __eq__(self, other):
 
     @classmethod
     def _csv_fields(cls):
-        return 'rate [ct./kWh]', 'energy [kWh]', 'price [ct.]', 'seller'
-
-    def _to_csv(self):
-        rate = round(self.energy_rate, 4)
-        return rate, self.energy, self.price, self.seller
+        return "rate [ct./kWh]", "energy [kWh]", "price [ct.]", "seller"
 
 
-def copy_offer(offer):
+def copy_offer(offer) -> Offer:
```

Comment:
What about renaming this method so that it will be the same for Offer and Bids? E.g.
```suggestion
def copy(offer) -> Offer:
# or
def create_duplicate(offer) -> Offer:
```